geotrellis-sentinelhub/src/test/scala/org/openeo/geotrellissentinelhub/PyramidFactoryTest.scala

```diff
@@ -1028,4 +1038,40 @@ class PyramidFactoryTest {
     verify(catalogApiSpy, atLeastOnce()).search(eqTo("sentinel-1-grd"), any(), eqTo(LatLng),
       eqTo(from), eqTo(ZonedDateTime.parse("2020-04-09T23:59:59.999999999Z")), any(), eqTo(util.Collections.emptyMap()))
   }
+
+  @Test
+  def testIssue128(): Unit = {
+    val endpoint = "https://services.sentinel-hub.com"
+
+    val catalogApiSpy = spy(new DefaultCatalogApi(endpoint))
+
+    val pyramidFactory = new PyramidFactory("sentinel-2-l2a", "sentinel-2-l2a", catalogApiSpy,
+      new DefaultProcessApi(endpoint), authorizer, sampleType = FLOAT32)
+
+    val polygons_crs = CRS.fromEpsgCode(32630)
+    val geojson = IOUtils.toString(getClass.getResource("/issue-128.geojson"))
```

Comment:
`IOUtils.toString` is deprecated.

The geojson file needs a more descriptive name.

Reply:
Instead of the `IOUtils.toString` workaround, would it be OK to move `ProjectedPolygons.fromVectorFile()` to `geotrellis-common` so that it can be used everywhere?

geojson renamed.

Reply:
I'm not sure what the implications are of making submodules dependent on each other but the code was simple enough to replace.

Reply:
The `GeoJson.parse` solution does not parse the CRS from the `.geojson`. But as we know the file beforehand, and it is in `LatLng` (as the standard demands), that is safe enough.